snow/consensus/snowstorm/conflicts/conflicts.go

```diff
@@ -412,27 +419,40 @@ func (c *Conflicts) Updateable() ([]Tx, []Tx) {
 
 			// Mark as rejectable txs that require transition [transitionID] to
 			// occur earlier than it can.
-			dependents := c.dependencies[transitionID]
-			for dependentID := range dependents {
-				dependent := c.txs[dependentID]
-				dependentEpoch := dependent.Epoch()
-				if dependentEpoch < lowestRemainingEpoch && !rejected.Contains(dependentID) {
-					rejected.Add(dependentID)
-					c.rejectable = append(c.rejectable, dependent)
+			for dependentTxID := range transitionNode.dependencies {
+				// If the transaction has already been rejected in this round
+				// then it must be skipped over since it has been removed from
+				// the txs map.
+				if rejected.Contains(dependentTxID) {
+					continue
+				}
```

Comment:
Added an extra check here to make sure that if a transaction has been marked as rejected it will not be checked again.